### PR TITLE
Update README.md for workflow

### PR DESCRIPTION
**Pull request by noahmehl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+This is a sub application of 2600hz/monster-ui.
+
+Users should either clone into monster-ui/src/apps/ such as:
```
Comment (azefiel) [inline README.md]:
```suggestion
Users should clone into `/monster-ui/src/apps/` such as:
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+This is a sub application of 2600hz/monster-ui.
```
Comment (azefiel) [inline README.md]:
```suggestion
# monster-ui-voip

This is a sub application of [2600hz/monster-ui](https://github.com/2600hz/monster-ui).
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+This is a sub application of 2600hz/monster-ui.
+
+Users should either clone into monster-ui/src/apps/ such as:
+
+$ cd monster-ui/src/apps/
+$ git clone https://github.com/2600hz/monster-ui-voip.git voip
```
[the remark this answers is not in the document]
Comment (noahmehl) [reply README.md]: @azefiel I have updated for this, please review :)

**Comment by azefiel** [comment]
Congrats 🎉 🎉 🎉 

Thanks again for your contribution!

**Pull request merged**